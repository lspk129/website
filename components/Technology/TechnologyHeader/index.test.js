import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import TehcnologyHeader from './';

describe('TehcnologyHeader', () => {
  it('renders corretcly', () => {
    const wrapper = renderer.create(<TehcnologyHeader />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders h2 element', () => {
    const wrapper = shallow(<TehcnologyHeader />);
    expect(wrapper).toBePresent();
    expect(wrapper).toHaveTagName('h2');
  });
});
