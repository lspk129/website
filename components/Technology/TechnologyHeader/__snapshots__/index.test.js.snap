// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TehcnologyHeader renders corretcly 1`] = `
<h2
  className="sc-bdVaJa hbgqAY"
/>
`;
